go/teams/chain.go

```diff
@@ -2248,6 +2277,48 @@ func (t *teamSigchainPlayer) parseTeamSettings(settings *SCTeamSettings, newStat
 	return nil
 }
 
+func (t *teamSigchainPlayer) parseTeamBotSettings(bots *[]SCTeamBot, newState *TeamSigChainState) error {
```

Comment:
dereferencing `bots` at the callsite would be a bit more normal. Btw afaik the slice struct (not the data) is tiny